librz/core/cfile.c

```diff
@@ -888,25 +889,39 @@ static bool map_multi_dex(RzCore *core, RzIODesc *desc, ut32 id) {
 	if (!rz_str_endswith(desc->name, ".dex")) {
 		return true;
 	}
+	RzCoreFile *cf = rz_core_file_cur(core);
+
+	// adds the current size and align next address
+	ut64 base_address = 0x200000;
+	RzBinFile *cur = rz_pvector_empty(&cf->binfiles) ? NULL : rz_pvector_tail(&cf->binfiles);
+	if (cur) {
+		RzIODesc *iod = rz_io_desc_get(core->io, cur->fd);
+		if (iod) {
```

Comment:
Maybe reverse the condition and switch to the "early exit" pattern?

Reply:
This condition cannot be reversed.